### PR TITLE
Clown Module feature addition.

### PR DESCRIPTION
**Pull request by PineappleWolf**
Adds Clown Module, Clown Borg Sprite, and transformation animation for Clown Borg Sprite from Standard.

HONK!

For when people want to be HONK but someone else took the job. They can now borg themselves and spread the HONK among the station.

Subject to feature changes/updates/new sprites.

### CODE REVIEW

**Comment by PineappleWolf** [comment]
<a href="http://imgur.com/8dgRoFz"></a>

Just imagine this space lubing you into hell.

**Comment by PineappleWolf** [comment]
Wait, did the sprite sheet go with it?

**Comment by PineappleWolf** [comment]
.dmi added, Borg can now be the clown.

**Comment by ShadowDeath6** [comment]
Maybe give it a second spray of colorful reagent because !FUN!

**Comment by PineappleWolf** [comment]
What is colorful reagent?

**Comment by ShadowDeath6** [comment]
Basically whatever it touches starts switching colors randomly. Also if it's ingested it makes you switch colors as long as it's in your system.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -286,6 +286,30 @@
 			modtype = "Jan"
 			feedback_inc("cyborg_janitor",1)
 
+	if("Clown")
+		var/icontype = input("Select an icon!", "Robot", "Clown") in list("Clown")
+		if(!icontype) return
+		module = new /obj/item/weapon/robot_module/clown(src)
+		hands.icon_state = "standard"
+		switch(icontype)
```
Comment (DavidEdwards) [inline code/modules/mob/living/silicon/robot/robot.dm:294]: Why is there a switch here? You only have one type of icon_state.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -286,6 +286,30 @@
...
+				icon_state = "disposalbot"
+				animation_length = 6
+			if("Chicken")
+				icon_state = "disposalbot"
+				animation_length = 6
+			else
+				icon_state = "janiborg"
+				animation_length = 22 */
+		modtype = "Jan"
+		feedback_inc("cyborg_janitor",1)
```
Comment (DavidEdwards) [inline code/modules/mob/living/silicon/robot/robot.dm]: This will need to be changed - it is no longer a janitor borg at this point.

**Comment by PineappleWolf** [comment]
It's a placeholder for future skins, it will have more sprites added,and I was using Janiborg to test the switch and the module. Also note it's commented out.

**Comment by PineappleWolf** [comment]
Realistically I can remove it for now, not a big deal, I'm just having it set up for me to shove in said "Wizard" and "Chicken" costume sprites.

**Comment by Kmc2000** [comment]
Why not my abominable sprite

reeee

**Comment by PineappleWolf** [comment]
I mean, I could add it, Kmc.

**Comment by PineappleWolf** [comment]
I also have this slightly off-centered abomination too, mine's based on standard, you'rs on a sec borg.

<a href="http://imgur.com/vCglDkr"></a>

**Comment by DavidEdwards** [comment]
I'd rather reduce the amount of code by removing those extra lines that don't really do anything. Down the line it may result in confusion.

**Comment by DavidEdwards** [comment]
Also, please make a changelog

**Comment by PineappleWolf** [comment]
How to change log

**Comment by DavidEdwards** [comment]
In the repository there is this folder:

html\changelogs\

Within that is example.yml

Simply copy that file, rename it and change the insides to support your change.

**Comment by PineappleWolf** [comment]
Yeah lemme know how that is

**Comment by PineappleWolf** [comment]
Yes I have an actual transformation animation for it based. 1 was just a test stage, the actual animation lasts about 8 frames, if I remember.

**Comment by PineappleWolf** [comment]
Wizard Bot hits the stage.

Edit: Borg to Bot.

<a href="http://imgur.com/muC6zXK"></a>

**Comment by PineappleWolf** [comment]
Wizard Borg hits the stage!

<a href="http://imgur.com/tGxScKm"></a>

**Comment by ghost** [comment]
this is stupid.

**Pull request merged**